src/main/scala/com/codacy/analysis/cli/tools/Tool.scala

```diff
@@ -132,9 +132,20 @@ class Tool(private val plugin: IDockerPlugin) {
 
 object Tool {
 
-  private val allTools = PluginHelper.dockerPlugins.++(PluginHelper.dockerEnterprisePlugins)
+  val allInternetToolShortNames: Set[String] =
```

Comment:
This this is only the internet tools we could call it just `internetToolShortNames` since this is not all tools.